Sasquatch/Sasquatch/ViewControllers/MSMainViewController.swift

```diff
@@ -103,6 +107,16 @@ class MSMainViewController: UITableViewController, AppCenterProtocol {
 
     // Make sure the UITabBarController does not cut off the last cell.
     self.edgesForExtendedLayout = []
+    
+    if (isUserSignedIn) {
+      authInfoTVCell.isUserInteractionEnabled = true
```

Comment:
TVCell? What is TVCell? Is it on TV? 📺

Reply:
It stands for TableViewCell I guess. 😝

Reply:
Sorry, it's just interim naming :sweat_smile: